go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/Azure/azure-sdk-for-go v63.1.0+incompatible
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.7.1
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.9.1
```

Comment:
Hmm, are the version bumps for `azure-sdk-for-go` needed for your unit test? I see that our current version provides `TokenCredential` as well, which I believe is what your test depends on.

Reply:
Hi Caden, I had to upgrade this to 1.9.1 for using fake in my unit tests.  I wanted to start using this as Petr once mentioned we should rely less on mocks and this is the official way to do it per the SDK.